src/github/middleware.ts

```diff
@@ -173,7 +172,8 @@ export default (
 
 			if (await booleanFlag(BooleanFlags.MAINTENANCE_MODE, false, jiraHost)) {
 				context.log(
-					`Maintenance mode ENABLED for jira host ${jiraHost} - Ignoring event of type ${webhookEvent}`
+					{jiraHost, webhookEvent},
```

Comment:
I've noticed you've done this in a couple of places. What's the advantage of moving away from string interpolation?

Reply:
In splunk or if you're using a log aggregate tool locally, it can group together errors with the same message and show you how often an issue actually happens.  This cannot happen when you use string interpolation since every message is now somewhat unique depending on the variable.  The message should be consistent across the board and the added benefit that if you put the variable in the 'data' of the log object, you can now filter by that variable.  No data is lost, but your logs improve drastically and makes it much easier to diagnose issues particularly at scale.

Reply:
I believe we should add jiraHost into context.log, when we know it, so it will be added to all logs automatically. (However this is out of scope of this PR)

Reply:
Oh that makes sense. Thanks for explaining @mboudreau